docs/technical_reference/unit_models/electrodialysis_1D.rst

```diff
@@ -0,0 +1,241 @@
+Electrodialysis (1D)
+====================
+
+Introduction
+------------
+
+Electrodialysis, an electrochemical separation technology, has been utilized to desalinate water for decades.
+Compared with other technologies, such as reverse osmosis (RO),
+electrodialysis shows advantages in desalinating brackish waters with
+moderate salinity by its less intense energy consumption, higher water recovery, and robust
+tolerance for adverse non-ionic components (e.g., silica and biological substances) in water.
+Between the electrodes of an electrodialysis stack (a reactor module), multiple anion- and
+cation-exchange membranes are alternately positioned and separated by spacers to form individual
+cells. When operated, electrodialysis converts electrical current to ion flux and, assisted by
+the opposite ion selectivity of cation- and anion-exchange membranes (cem and aem), moves ions from
+one cell to its adjacent cell in a cell-pair treatment unit (Figure 1). The ion-departing cell is called a **diluate
+channel** and the ion-entering cell a **concentrate channel**. Recovered (desalinated) water is
+collected from diluate channels of all cell pairs while the concentrate product can be disposed of as brine
+or retreated. More overview of the electrodialysis technology can be found in the *References*.
+
+.. figure:: ../../_static/unit_models/EDdiagram.png
+    :width: 600
+    :align: center
+
+    Figure 1. Schematic representation of an electrodialysis cell pair
+
+
+One cell pair in an electrodialysis stack can thus be treated as a modeling unit that can multiply to
+larger-scale systems.  The presented electrodialysis 1D model establishes mathematical descriptions of
+ion and water transport in a cell pair and expands to simulate a stack with a specified cell-pair number.
+Modeled mass transfer mechanisms include electrical migration and diffusion of ions and osmosis and electroosmosis
+of water. By its name, "1D" suggests the variation of mass transfer and solution properties over length (l in Figure 1)
+is mathematically described.  The model relies on the following key assumptions:
+
+* The concentrate and diluate channels have identical geometry.
+* For each channel, component fluxes in the bulk solution are uniform in b and s directions in Figure 1
+  but dependent on the l directions (the 1-dimensional assumption).
+* Steady state: all variables are independent of time.
+* Co-current flow operation. 
+* Electrical current is operated below the limiting current. 
+* Ideality assumptions: activity, osmotic, and van't Hoff coefficients are set at one.
```

Comment:
Is the ideality assumption necessarily applicable? The ion prop model has the Davies model, so at the least, activity can deviate from ideality assumptions. Is there something in the ED unit model that would need to be incorporated, beyond what the ion prop model provides (in the case, just referring to the activity)? If not, and the ED model can technically utilize activities calculated by the Davies model (or might be that you'd need to apply activity coefficients--forgot how it was implemented in prop model), then you can boast that the ED model _can _ account for nonideality in terms of activities. My presumption is that you only tested the model with activity coefficients = 1 but the model would work if provided non-ideal activity coefficients. Is that the case?